src/Wallabag/CoreBundle/Controller/SearchController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Wallabag\CoreBundle\Controller;
+
+use Pagerfanta\Adapter\DoctrineORMAdapter;
+use Pagerfanta\Pagerfanta;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Wallabag\CoreBundle\Entity\ArticleSearch;
+use Wallabag\CoreBundle\Form\Type\EntryFilterType;
+use Wallabag\CoreBundle\Form\Type\EditEntryType;
+use Wallabag\CoreBundle\Form\Type\NewSearchType;
+
+use Elastica\Index;
+use Elastica\Query;
+use Elastica\Query\Term;
+use Pagerfanta\Adapter\ElasticaAdapter;
+use Pagerfanta\Adapter\ArrayAdapter;
+
+class SearchController extends Controller
+{
+    /**
+     * @param Request $request
+     *
+     * @Route("/search/{page}", name="search", defaults={"page" = "1"})
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function searchFormAction(Request $request, $page = 1) {
+
+        $articleSearch = new ArticleSearch();
+
+        $form = $this->createForm(NewSearchType::class,$articleSearch);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+
+            $repositoryManager = $this->get('fos_elastica.manager.orm');
+            $repository = $repositoryManager->getRepository('WallabagCoreBundle:Entry');
+            $search = $articleSearch->getSearchTerm();
+            $logger = $this->get('logger');
+            $logger->error($search);
+
+            $articles = $repository->find($search);
```

Comment:
By doing this, you make use of the QueryString clause (https://github.com/ruflin/Elastica/blob/master/lib/Elastica/Query.php#L84) I think.

This is not a good way to search in Elasticsearch,
as you are using:
- the `_all` field by default, which is a concatenation of everything, analyzed in standard
- you look in ALL the values without type consideration, so dates for example are messed up
- this is very sensible to syntax error and throw exception if the parse fails
- it is possible to forge very costly QueryString search...

Please consider building your own request using the DSL.